ci/build/build-standalone-release.sh

```diff
@@ -28,8 +28,31 @@ main() {
   ln -s "./lib/node" "$RELEASE_PATH/node"
 
   pushd "$RELEASE_PATH"
-  yarn --production --frozen-lockfile
+  yarn --frozen-lockfile
+  create_shrinkwraps
   popd
 }
 
+create_shrinkwraps() {
```

Comment:
The `build-standalone-release.sh` script takes the npm package generated in a previous step, installs deps, then packages the result so we can upload a standalone build complete with deps to GitHub.  Assuming I have that right (sometimes I get confused with all the steps) generating a shrinkwrap here would not get uploaded to npm.

So I think we will have to do this in `build-release.sh` which is where we create and upload the npm package artifact that eventually gets uploaded to npm.

And then here we would install with `npm` instead of `yarn`.

Reply:
Ah damn yes, now I think I remember being bugged/blocked by something like that in the past... Gonna double-check that now!

Reply:
> And then here we would install with `npm` instead of `yarn`.

Can I suggest to avoid doing too many things in the same PR if that works for you:
* (This PR) Just adding the `npm-shrinkwrap.json` file - which basically should not change anything else in the development/CI workflow.
* Another PR to change the `release:standalone` and other CI scripts to start using `npm ci`
* Another PR (when everything works) to amend the doc to recommend `npm` (which can be done last, as using yarn will still always work).

Reply:
I like the incremental approach!